ang/civicase/activity/feed/directives/activity-feed.directive.js

```diff
@@ -484,6 +483,13 @@
       $scope.$on('civicase::activity-feed::hide-activity-panel', function () {
         toggleMonthNavVisibility(true);
       });
+      $scope.$watch('params.filters', function () {
+        if ($scope.params && $scope.params.filters) {
```

Comment:
This if condition already exists. Lets move this to a function like `applyFilters` or `extendFilters`